cogment_verse/run/run_session.py

```diff
@@ -24,7 +26,7 @@
 
 
 class RunSession:
-    def __init__(self, run_cfg, run_id, services_directory, model_registry):
+    def __init__(self, run_cfg, run_id, services_directory, model_registry: ModelRegistry):
```

Comment:
Even though I like the idea of adding object types, in the cogment-verse it looks like an inconsistent coding style.

Reply:
I agree it won't be consistent for now but I think we should move towards the hint type in order to facilitate the code navigation using one click instead of searching option, especially for new users.